Makefile

```diff
@@ -14,7 +14,7 @@ conda:
 	set -ex
 	conda env create -f environment.yml
 	conda activate nams
```

Comment:
CommandNotFoundError: Shell not configured to use 'conda activate'.

$(conda activate nams) resolved the error.

Reply:
@mirbehroznoor Which shell is this, and which platform?

Reply:
Ubuntu 20.04.2 LTS focal, the shell is bash. 

Worked:
I did setup everything with Emacs "elpy-rpc-virtualenv" python 3.8 and bash shell,  with ```$(conda activate nams)```.

Failed:
However, with system python ```/usr/bin/python``` Makefile raised this issue:
```bash
make conda
set -ex
conda env create -f environment.yml

python setup.py develop
running develop
error: can't create or remove files in install directory

The following error occurred while trying to add or remove files in the
installation directory:

    [Errno 13] Permission denied: '/usr/local/lib/python2.7/dist-packages/test-easy-install-4557.write-test'

The installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:

    /usr/local/lib/python2.7/dist-packages/

Perhaps your account does not have write access to this directory?  If the
installation directory is a system-owned directory, you may need to sign in
as the administrator or "root" account.  If you do not have administrative
access to this machine, you may wish to choose a different installation
directory, preferably one that is listed in your PYTHONPATH environment
variable.

For information on other options, you may wish to consult the
documentation at:

  https://setuptools.readthedocs.io/en/latest/easy_install.html

Please make the appropriate changes for your system and try again.

make: *** [Makefile:19: conda] Error 1
```
Also Worked:
Converting the Makefile into a bash script with some changes did the trick even with ```/usr/bin/python``` in setting up ```conda env```.